models/enterprise_sonic/lldp_interfaces/sonic_lldp_interfaces.yml

```diff
@@ -0,0 +1,103 @@
+        'license': 'Apache 2.0'
+    }
+NETWORK_OS: sonic
+RESOURCE: lldp_interfaces
+COPYRIGHT: Copyright 2022 Dell Inc. or its subsidiaries. All Rights Reserved
+DOCUMENTATION: |
+    module: sonic_lldp_interfaces
+    version_added: '2.1.0'
+    short_description: Manage Inteface LLDP configurations on SONiC
+    description:
+      - This module provides configuration management of interface LLDP parameters
+        in devices running SONiC.
+        It is intended for use in conjunction with global LLDP.
+    author: 'Divya Balasubramanian(@divya-balasubramania)'
+    options:
+      config:
+        description: The set of link layer discovery protocol interface attribute configurations
+        type: list
+        elements: dict
+        suboptions:
+          name:
+            description:
+              - Interface name in which LLDP needs to be configured on.
+            type: str
+            required: True
+          enable:
+            description:
+              - This argument is a boolean value to enable or disable LLDP.
+              - This command is supported only on physical interfaces and not on logical interfaces.
+            type: bool
+          mode:
+            description:
+              - By default both transmit and receive of LLDP frames is enabled.
+              - This command can be used to configure either in receive only or transmit only mode.
+              - This command is supported on physical and logical interfaces.
+            type: str
+            choices:
+               - 'receive'
+               - 'transmit'
+          med_tlv_select:
+            description:
+              - This command can be used to select whether to advertise the LLDP-MED TLVs or not.
+                By default the LLDP-MED TLVs are advertised.
+              - This command is supported only on physical interfaces and not on logical interfaces.
+            type: list
+            elements: dict
+            suboptions:
+              network_policy:
+                description:
+                     - This command can be used to select whether to advertise network-policy
+                       LLDP-MED TLVs or not.By default network-policy LLDP-MED TLVs are advertised.
+                type: str
+              power_management:
+                description:
+                     - This command can be used to select whether to advertise power-management
+                       LLDP-MED TLVs or not.By default power-management LLDP-MED TLVs are advertised.
+                type: str
+          tlv_select:
+             description:
+               - This command can be used to select whether to advertise the LLDP 802.3at or bt
+                 power management TLVs or not. By default this TLV is advertised.
+               - This command is supported only on physical interfaces and not on logical interfaces.
+             type: str
+             choices:
+                - 'power-management'
```

Comment:
Please change "tlv_select" to `list` and "power_management" as one of its sub-options of type `bool`, similar to structure of "med_tlv_select"